### DISCUSSION TITLE
How to achieve the mica effect (frosted glass or acrylic effect) on the Linux platform?

### DISCUSSION THREAD

**Discussion by c61292558** [fltk-rs/fltk-rs#1430] [Q&A]
I think the window mica (acrylic) effect on Win10/11 (as shown below) is very good-looking. How can it be achieved?
![image](https://github.com/fltk-rs/fltk-rs/assets/11317413/6aef987b-d5d1-4b30-ab93-bcf88d3fbb0c)
![image](https://github.com/fltk-rs/fltk-rs/assets/11317413/e8d29f36-bef6-4f3d-9d5e-8cf3e332e5ec)
This effect gives people a very calm feeling, it is very unique.
Thanks for your help.

**Comment by MoAlyousef** [comment]
I’m not sure. Usually these require calls to platform specific apis to achieve such effects. See https://github.com/tauri-apps/window-vibrancy which also only works on windows and macos.

**Reply by c61292558** [reply]
thank you.Then I can only switch to other effects.
